lsp-methods.el

```diff
@@ -1731,19 +1706,16 @@ Returns xref-item(s)."
     (lsp--send-notification (lsp--make-notification "$/cancelRequest"
                               `(:id ,id)))))
 
-(defun lsp--on-hover ()
+(defun lsp-eldoc-function ()
   ;; This function is used as ‘eldoc-documentation-function’, so it’s important
   ;; that it doesn’t fail.
-  (with-demoted-errors "Error in ‘lsp--on-hover’: %S"
-    (when (and (lsp--capability "documentHighlightProvider")
-               lsp-highlight-symbol-at-point)
-      (lsp-symbol-highlight))
-    (when (and (or (lsp--capability "codeActionProvider")
-                   (lsp--registered-capability "textDocument/codeAction"))
-               lsp-enable-codeaction)
-      (lsp--text-document-code-action))
```

Comment:
lsp--text-document-code-action is no longer used, do you have a plans to do something with it?